portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,7 +58,7 @@ private String getComment(HttpServletRequest request) {
     //   Get comment from form.
     String comment = request.getParameter("comment");
     if (comment == null) {
```

Comment:
We shouldn't add new comments if the request is malformed in some way. Could the method return null and allow the caller to send an error back to the client?

Reply:
Sure i'll try